### ISSUE TITLE
Is there a way to take multiple inner steps in your code?

### ISSUE THREAD

**Issue by chnsh** [tristandeleu/pytorch-maml#10]
I wanted to take multiple inner gradient steps but wasn't sure how to do that in Torchmeta.

Thanks!

**Comment by tristandeleu** [comment]
Yes you can take multiple inner gradient steps with this implementation. Using the `train.py` script, you can specify the number of gradient steps with `--num-steps`.
https://github.com/tristandeleu/pytorch-maml/blob/44104272a0140b35e2223ba68750e7e715315653/train.py#L127-L129

Or internally in the `ModelAgnosticMetaLearning` class, this is the `num_adaptation_steps` argument.
https://github.com/tristandeleu/pytorch-maml/blob/44104272a0140b35e2223ba68750e7e715315653/maml/metalearners/maml.py#L40-L42
